indexing-hadoop/src/main/java/io/druid/indexer/IndexGeneratorJob.java

```diff
@@ -448,11 +448,12 @@ public void setConf(Configuration config)
 
     protected ProgressIndicator makeProgressIndicator(final Context context)
     {
-      return new BaseProgressIndicator()
+      return new LoggingProgressIndicator("IndexGeneratorReducer")
```

Comment:
this will cause regression, pls see https://github.com/druid-io/druid/pull/1923 .